README.md

```diff
@@ -23,6 +23,26 @@ Windows (not actively maintained as a development platform)
    on a Linux computer. Otherwise, run `id -u` and `id -g` and use the resulting numbers in place of `1000`.
 5. Run `make start`, or if using Vagrant, run `vagrant up`
 
+## Configuration
+By default, configuration is read from environment variables. These are documented
+in the `local.env.dist` file. Optionally, you can define configuration in AWS AppConfig.
+To do this, set the following environment variables to point to the configuration in
+AWS:
+
+* `AWS_REGION` - the AWS region in use
+* `APP_ID` - the application ID or name
+* `CONFIG_ID` - the configuration profile ID or name
+* `ENV_ID` - the environment ID or name
+
+In addition, the AWS PHP SDK requires authentication. It is best to use an access role
```

Comment:
Are we using the AWS PHP SDK? I thought that config-shim provided the necessary values as environment variables, and the PHP code wasn't interacting directly with AWS AppConfig... though admittedly my understanding of this is still thin.

Reply:
The AWS API is in use by the shim, but not by idp-id-broker PHP code. So you raise a valid point, that the PHP SDK is not being used. I'll make a correction to this.